components/announcement_bar/configuration_bar/configuration_bar.jsx

```diff
@@ -137,13 +137,19 @@ export default class ConfigurationAnnouncementBar extends React.PureComponent {
             let defaultMessage;
             if (this.props.config.EnableSignUpWithGitLab === 'true') {
                 id = t('announcement_bar.error.site_url_gitlab.full');
-                defaultMessage = 'Please configure your [Site URL](https://docs.mattermost.com/administration/config-settings.html#site-url) in the [System Console](/admin_console/environment/web_server) or in gitlab.rb if you\'re using GitLab Mattermost.';
+                defaultMessage = 'Please configure your [Site URL](https://docs.mattermost.com/administration/config-settings.html#site-url) in the [System Console]({basename}/admin_console/environment/web_server) or in gitlab.rb if you\'re using GitLab Mattermost.';
             } else {
                 id = t('announcement_bar.error.site_url.full');
-                defaultMessage = 'Please configure your [Site URL](https://docs.mattermost.com/administration/config-settings.html#site-url) in the [System Console](/admin_console/environment/web_server).';
+                defaultMessage = 'Please configure your [Site URL](https://docs.mattermost.com/administration/config-settings.html#site-url) in the [System Console]({basename}/admin_console/environment/web_server).';
             }
 
-            const siteURLMessage = formatMessage({id, defaultMessage});
+            let basename = window.basename;
```

Comment:
Can we avoid using `window.basename` (i.e. the global) directly? And instead bind the result of `getBasePath` into the component?